src/bench/bench.ts

```diff
@@ -5,16 +5,17 @@ import { Readable } from "stream";
 import { log } from "../utils";
 import { json } from "./json";
 import { ChildProcessResult, MARKER } from "./protocol";
+import findRoot from "find-root";
 
 type ChildProcess = ChildProcessByStdio<null, Readable, null>;
 
 function monitorLogs(childProcess: ChildProcess): Promise<ChildProcessResult> {
   const childStdOut = childProcess.stdout;
-  return new Promise((resolve, reject) => {
+  return new Promise((resolve) => {
     const onEnd = () => {
       childStdOut.removeListener("data", onData);
       childStdOut.removeListener("end", onEnd);
-      return reject("Failed to find metric output line in child process. Did it terminate correctly?");
+      throw "Failed to find metric output line in child process. Did it terminate correctly?";
```

Comment:
using `reject` creates stackless error messages of unhandled rejected promise. With `throw` the error bubbles up to its `await`.

Reply:
The wonders and joy of JS.

Reply:
Also `throw new Error("...")` is a best practice for this kind of thing. If you throw a string, you also catch a string, whereas if you throw and error, you get an error object with a stack property and other useful stuff. Surprised there is no lint rule on for this, probably could be added to the gadget base lint config.